config/yugabyte/regression_pipelines/scan_workloads/yugabyte/scanG11_pkey_rangeScan_INclause_project_indexed_nonindexed_cols.yaml

```diff
@@ -0,0 +1,127 @@
+#TESTS:
+#   Range scan time to fetch 1 rows
+#   Range scan time to fetch 10 rows
+#   Range scan time to fetch 100 rows
+#   Range scan time to fetch 1000 rows
+#   Range scan time to fetch 10000 rows
+type: YUGABYTE
+driver: com.yugabyte.Driver
+url: jdbc:yugabytedb://{{endpoint}}:5433/yugabyte?sslmode=require&ApplicationName=featurebench&reWriteBatchedInserts=true&load-balance=false
+username: {{username}}
+password: {{password}}
+batchsize: 128
+isolation: TRANSACTION_REPEATABLE_READ
+loaderthreads: 1
+terminals: 1
+collect_pg_stat_statements: true
+use_dist_in_explain : true
+works:
+    work:
+        time_secs: 120
+        active_terminals: 1
+        rate: unlimited
+        warmup: 30
+microbenchmark:
+    class: com.oltpbenchmark.benchmarks.featurebench.customworkload.YBDefaultMicroBenchmark
+    properties:
+        setAutoCommit: true
+        create:
+            - DROP TABLE IF EXISTS pkey_rangescan_for_INclause_tbl_1;
+            - CREATE TABLE pkey_rangescan_for_INclause_tbl_1(col_bigint_id_1 bigint, col_bigint_1 bigint, col_bigint_2 bigint, col_bigint_3 bigint, col_bigint_4 bigint, col_bigint_5 bigint, col_float2_1 float(2), col_float2_2 float(2), col_float5_1 float(5), col_float5_2 float(5), col_boolean_1 boolean, col_varchar10_1 varchar(10), col_varchar100_1 varchar(100), col_varchar100_2 varchar(100), col_varchar500_1 varchar(500), PRIMARY KEY(col_bigint_id_1 ASC));
+        cleanup:
+            - DROP TABLE IF EXISTS pkey_rangescan_for_INclause_tbl_1;
+        loadRules:
+            - table: pkey_rangescan_for_INclause_tbl_
+              count: 1
+              rows: 1000000
+              columns:
+                  - name: col_bigint_id_
+                    count: 1
+                    util: PrimaryIntGen
+                    params: [1, 1000000]
+                  - name: col_bigint_
+                    count: 5
+                    util: RandomNumber
+                    params: [1, 1000000]
+                  - name: col_float2_
+                    count: 2
+                    util: RandomNoWithDecimalPoints
+                    params: [1, 1000000, 2]
+                  - name: col_float5_
+                    count: 2
+                    util: RandomNoWithDecimalPoints
+                    params: [1, 1000000, 5]
+                  - name: col_boolean_
+                    count: 1
+                    util: RandomBoolean
+                  - name: col_varchar10_
+                    count: 1
+                    util: RandomAString
+                    params: [10, 10]
+                  - name: col_varchar100_
+                    count: 2
+                    util: RandomAString
+                    params: [100, 100]
+                  - name: col_varchar500_
+                    count: 1
+                    util: RandomAString
+                    params: [500, 500]
+        executeRules:
+            - workload: scanG13_pkey_INclause_rangescan_increasing_rows_1row
```

Comment:
The workload name prefix should also be changed from scanG13 to scanG11 for all workloads in all 3 files.